### ISSUE TITLE
Unable to define SectionList using name of Plex library

### ISSUE THREAD

**Issue by randybruder** [ngovil21/Plex-Cleaner#4]
In the configuration for `SectionList`, I wanted to target only my Plex library `TV Shows (To Watch)`, so I set it to:
```python
SectionList = ["TV Shows (To Watch)"]  # Sections to clean. If empty all sections will be looked at
```

When running the script, it errors and doesn't run. Here's a snippet of the output:
```
Host: http://127.0.0.1
Port: 32400
Section List Mode: User-defined
Operating on user-defined sections: TV Shows (To Watch)
HTTP Error 400: Bad Request
HTTP Error 400: Bad Request
HTTP Error 400: Bad Request
HTTP Error 400: Bad Request
HTTP Error 400: Bad Request
HTTP Error 400: Bad Request
Failed to load Section TV Shows (To Watch). Skipping...
```

After a bit of tinkering, I found that by going to `http://127.0.0.1:32400/library/sections` on my Plex server, in the returned XML file finding the library, and noting the `key` attribute. Here's a screenshot:

![Screenshot of the XML file](http://i.imgur.com/lte2Q5c.png)

I used that value instead:

```python
SectionList = [1]  # Sections to clean. If empty all sections will be looked at
```

and the script runs successfully. Here's a snippet of the output:

```
Host: http://127.0.0.1
Port: 32400
Section List Mode: User-defined
Operating on user-defined sections: 1

--------- Section 1: TV Shows (To Watch) -----------------------------------
```

If this is the case (if you're required to use the `key` attribute rather than the name of the library), it should be documented in the project readme. Or, if I'm doing something wrong, let me know the proper way to define `SectionList`.

Thanks for making a great Python script!

**Comment by ngovil21** [comment]
Ya it's set to run by key, that's more for convenience in the script because the url uses the key. I'll add it to the documentation and look into adding sections by title.

**Comment by randybruder** [comment]
I haven't used Python before, but code is code, so I took a stab at rewriting the script to resolve this.

The two places I changed code:

If `SectionList` isn't defined, the script will check every `key` against `IgnoreSections`. I updated it to also check every `title` against  `IgnoreSections` as well:

```python
if Section.getAttribute("key") not in Settings['IgnoreSections'] and Section.getAttribute("title") not in Settings['IgnoreSections']:
```

And secondly, when `SectionList` is defined, I make a new array called `section_keys`, and I check every Section to see if either the `title` or `key` is in `SectionList`.

If either are, I add the `key` to the array `section_keys`. Then I 'reset' the `SectionList` to the `section_keys`. Essentially I'm filtering out any text-based sections called, and replacing them with the key value.

```python
log("Section List Mode: User-defined")
    section_keys = []
    for Section in doc_sections.getElementsByTagName("Directory"):
        if Section.getAttribute("title") in Settings['SectionList'] or Section.getAttribute("key") in Settings['SectionList']:
            section_keys.append(Section.getAttribute("key"))
    
    Settings['SectionList'] = section_keys
    log("Operating on user-defined sections: " + ','.join(str(x) for x in Settings['SectionList']))
```

I'll make a pull request in a minute with these changes. I've tested the script and it works as intended for me, but I'd like to stress that I don't know Python, and my changes should be thoroughly vetted.

Again thanks for a fantastic script, and glad to hopefully have helped!
